modules/xmpp/strophe.jingle.js

```diff
@@ -345,23 +349,7 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
             case 'turn':
             case 'turns': {
                 dict.urls = `${type}:`;
-                const username = el.attr('username');
-
-                // https://code.google.com/p/webrtc/issues/detail
-                // ?id=1508
-
-                if (username) {
-                    const match
-                        = navigator.userAgent.match(
-                            /Chrom(e|ium)\/([0-9]+)\./);
-
-                    if (match && parseInt(match[2], 10) < 28) {
-                        dict.urls += `${username}@`;
```

Comment:
@fippo This is correct right, this was some workaround for chrome 27 and below which we no longer need :)? 

The supported chrome version was bumped to 72 recently.

Reply:
kill it with :fire: